lib/happymapper/item.rb

```diff
@@ -146,15 +139,9 @@ def convert_string_to_constant(type)
       names = type.split('::')
       constant = Object
       names.each do |name|
-        constant =
-          if constant.const_defined?(name)
-            constant.const_get(name)
-          else
-            constant.const_missing(name)
-          end
+        constant = constant.const_defined?(name) ? constant.const_get(name) : constant.const_missing(name)
```

Comment:
I'm not too sure about this one either.

What is your rationale for preferring `? :` over plain `if .. else .. end`?